cadence/contracts/Shard.cdc

```diff
@@ -36,7 +36,11 @@ pub contract Shard: NonFungibleToken {
         pub let splits: UInt8
 
         // The metadata for a Moment
-        pub let metadata: {String: String}
+        access(self) let metadata: {String: String}
+
+        pub fun getMetadata(): {String: String} {
```

Comment:
I think if you declared `metadata` as `access(contract)` you wouldn't need this?  https://docs.onflow.org/cadence/language/access-control/

Reply:
7693dde